src/pages/Projects/index.js

```diff
@@ -157,7 +160,12 @@ export default function () {
           projectComponents.map((component, key) => {
             return (
               <Grid.Column key={key}>
-                <Card onClick={() => setOpen(key)} style={cardStyle} className='card-styling'>
+                <Card onClick={() => {
+                  // required to fix button
+                  if (openKey !== key) setOpen(key);
```

Comment:
### IMPORTANT FIX:
For some reason, onClick propagates to its child components, including the modal it opens. So when clicking on the close icon of the open modal, it trigger's the **Card's** onClick callback **after** firing the **Modal's** onClose callback forcing the modal to stay open. To avoid this, we only allow the **Card's** onClick to fire if it is not already open.